nixos/modules/services/networking/sabnzbd/config_merge.py

```diff
@@ -0,0 +1,19 @@
+import sys
+import os.path
+from configobj import ConfigObj, ParseError
+
+base = ConfigObj()
+
+for path in sys.argv[1:]:
+    if os.path.isfile(path):
+        with open(path) as f:
+            try:
+              update = ConfigObj(f)
+              base.merge(update)
+            except ParseError as e:
+              raise Exception(f"Failed to merge {path}") from e
```

Comment:
Should we not expose the error?

Reply:
python's `from` syntax will wrap and expose the error:

```python
>>> raise RuntimeError("Splines disreticulated") from Exception("Critical gravitas shortage detected")
Exception: Critical gravitas shortage detected

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
RuntimeError: Splines disreticulated
```